src/main/scala/org/constellation/http/Loadbalancer.scala

```diff
@@ -80,6 +80,17 @@ class Loadbalancer(port: Int = 9000, host: String = "localhost", retryAfterMinut
         .map(_.size)
         .flatTap(size => logger.info(s"I understand Im healthy, my upstream size is now $size"))
         .flatMap(size => Ok(s"$size"))
+    case GET -> Root / "health" / addrStr =>
+      Addr
+        .unapply(addrStr)
+        .map { addr =>
+          upstream.get.flatMap(
+            _.find(_ == addr)
```

Comment:
Eventually `_.exists` or `_.existsM` if you ignore returned value anyway

Reply:
I don't know if I got your idea but the other way I can write it is `      Addr
        .unapply(addrStr)
        .map { addr =>
          upstream.get.flatMap {
            case i if i.contains(addr) => NoContent()
            case _ => NotFound()
          }
        }
        .getOrElse(BadRequest())` and I don't really have a strong opinion what's better.

Reply:
Ah ok, nevermind. I mislooked the type